.gitlab-ci.yml

```diff
@@ -0,0 +1,10 @@
+generate ci jobs:
+  script:
+    - "./bin/generate-gitlab-ci-yml.sh"
+  tags:
+    - "spack-k8s"
+  image: "scottwittenburg/spack_builder_ubuntu_18.04"
```

Comment:
At some point, this should be update to `spack/ci:ubuntu-18.04`.  See #11621.

Reply:
Yep, makes sense.  I'll take a look at that some time today and provide some feedback as well.  If we can get that merged and the images pushed to Dockerhub, I could fix this and have one less thing to wait around for.